### ISSUE TITLE
MqttClient generateClientId

### ISSUE THREAD

**Issue by AnilamSamoht** [fr3ts0n/AndrOBD-Plugin#7]
Hi,
everytime I start this application, its generate new ClientId for the MqttClient. Is there a possibility to generate an ClientID at firs run of this plugin or to set it up in the preferences of this plugin? I'm not able to do it by my self :-(

**MqttPlugin.java:**
```
171 // set a proper client id if we have none
172		if (mClientId == null || mClientId.trim().equals(""))
173		{
174			mClientId = MqttClient.generateClientId();
175		}
176		updateThread.start();
```

**Comment by fr3ts0n** [comment]
Thank you for your report.

I'll fix that and make the client id persistent, and adjustable in the preferences.

**Comment by fr3ts0n** [comment]
Fix is available now for your tests on [AndrOBD dev channel](https://t.me/AndrOBD_dev)

- MqttPublisher plugin V1.0.3

**Comment by AnilamSamoht** [comment]
Thanx. It works now.

**Comment by fr3ts0n** [comment]
released with MqttPublisher V1.0.3
